nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] **Good job so far 👏 :clap:** However, it would be better to provide **informative error messages** when raising `NotImplementedError` exceptions. This can help with debugging and make it easier for developers to understand the cause of the error. 👍 👍 

example:
```
raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"